.github/workflows/container-build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build container image every change.
+
+on:
+  workflow_call:
+    outputs:
+      digest:
+        description: "Container image digest"
+        value: ${{jobs.build.outputs.digest}}
+  push:
+    branches:
+      - "*"
```

Comment:
I think we should limit this to `main`. If not, the `Retrieve tag name` step of the other workflow, the container-image.yml one, will overwrite the `:latest` tag with whatever build from whatever branch that has run the workflow the latest.

Reply:
That's true. Maybe we can do something similar to what we have in the controller. Thus, we can have a container image for each feature branch. Which is nice for testing and validation of our changes.

Reply:
this should be solved now